Assets/_Project/Code/Scripts/Badges/BadgesController.cs

```diff
@@ -0,0 +1,10 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class BadgesController : MonoBehaviour
+{
+    public void ViewMenu()
```

Comment:
View sugeruje, że to jakieś okienko albo popup, proponuje GoToMainMenu albo LoadMainMenuScene